firebase-installations/src/main/java/com/google/firebase/installations/local/IidStore.java

```diff
@@ -86,14 +86,23 @@ private static String getDefaultSenderId(FirebaseApp app) {
     return projectNumber;
   }
 
-  private String createTokenKey(@NonNull String senderId) {
-    return STORE_KEY_TOKEN + senderId + DEFAULT_SCOPE;
+  private String createTokenKey(@NonNull String senderId, @NonNull String scope) {
+    return STORE_KEY_TOKEN + senderId + " | " + scope;
   }
 
   @Nullable
   public String readToken() {
     synchronized (iidPrefs) {
-      String token = iidPrefs.getString(createTokenKey(defaultSenderId), null);
+      Log.d("Rayo", "readToken: defaultSenderId: " + defaultSenderId);
```

Comment:
I know you would have removed it, but still annoying you with a comment, haha

Reply:
done